<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SONY-VAIO" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="14 Feb 2019 20:00:02 GMT" time="0.062" errors="0">
  <testcase name="test3" time="0.016" classname="com.selenium_maven.testng_retry">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.selenium_maven.testng_retry.test3(testng_retry.java:25)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test2" time="0.015" classname="com.selenium_maven.testng_retry"/>
  <testcase name="test1" time="0.0" classname="com.selenium_maven.testng_retry">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
